### ISSUE TITLE
Basal area chart for open loop not consistent

### ISSUE THREAD

**Issue by astralis1** [Artificial-Pancreas/iAPS#769]
I've installed the app in the past 24 hours, only closing the loop for short bursts 2-3 times. Now with open loop, the blue area chart appears to be inconsistent. (see photo). I assume it represents the basal. 

![IMG_3158](https://github.com/Artificial-Pancreas/iAPS/assets/175309242/8975a7a7-3549-4072-b2fe-712e6976b8b3)

**Comment by Jon-b-m** [comment]
Yes, thanks. This is an old issue since FreeAPS X. I’ll look into it very soon.
